lib/crypto/ec/Secp256k1CryptoSuite.ts

```diff
@@ -0,0 +1,65 @@
+import EcPublicKey from './EcPublicKey';
+import CryptoSuite from '../../interfaces/CryptoSuite';
+import PrivateKey from '../../security/PrivateKey';
+import PublicKey from '../../security/PublicKey';
+import { DidPublicKey } from '@decentralized-identity/did-common-typescript';
+
+const ecKey = require('ec-key');
+
+/**
+ * Encrypter plugin for Elliptic Curve P-256K1
+ */
+export class Secp256k1CryptoSuite implements CryptoSuite {
+  /** Encryption with Secp256k1 keys not supported */
+  getEncrypters () {
+    return {};
+  }
+
+  getSigners () {
+    return {
+      P256K: {
+        sign: Secp256k1CryptoSuite.sign,
+        verify: Secp256k1CryptoSuite.verify
+      }
+    };
+  }
+
+  getKeyConstructors () {
+    return {
+      Secp256k1VerificationKey2018: (keyData: DidPublicKey) => {

```

Comment:
Shouldn't this be `EdDsaSAPublicKeySecp256k1` as per [EdDsaSASignatureSecp256k1](https://w3c-ccg.github.io/ld-cryptosuite-registry/)?

Reply:
There are several in use in the wild. Adding support for multiple including the ones above as described in [Linked Data Cryptographic Suite Registry](https://w3c-ccg.github.io/ld-cryptosuite-registry/#eddsasasignaturesecp256k1)